### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /notebook-ci-testing/notebooks/NIRCAM/Coronagraphy/JWPipeNB-nircam-coronagraphy.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#240]
## What
An accessibility scan flagged 3 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/NIRCAM/Coronagraphy/JWPipeNB-nircam-coronagraphy.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following elements need attention:

- `<a class="reference external" href="https://jwst-pipeline.readthedocs.io/en/latest/api/jwst.jump.JumpStep.html"><code class="docutils literal notranslate"><span class="pre">jump</span></code> step</a>` (selector: `.alert-info > p > .external.reference:nth-child(2)`)
- `<a class="reference external" href="https://jwst-docs.stsci.edu/known-issues-with-jwst-data/shower-and-snowball-artifacts">snowballs</a>` (selector: `.alert-info > p > .external.reference:nth-child(4)`)
- `<a class="reference external" href="https://www.stsci.edu/files/live/sites/www/files/home/jwst/documentation/technical-documents/_documents/JWST-STScI-008545.pdf">Detection and Flagging of Showers and Snowballs in JWST Technical Report (Regan 2023)</a>` (selector: `.external.reference:nth-child(7)`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 4.1 (foreground color: #176de8, background color: #cff4fc, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
